comps/dataprep/neo4j/llama_index/README.md

```diff
@@ -0,0 +1,94 @@
+# Dataprep Microservice with Neo4J
+
+This dataprep microservice ingests the input files and uses LLM (TGI or OpenAI model when OPENAI_API_KEY is set) to extract entities, relationships and descriptions of those to build a graph-based text index.
+
+### Setup Environment Variables
+
+```bash
+#Manually set private environment settings
+export host_ip=${your_hostname IP} #local IP
+export no_proxy=$no_proxy,${host_ip} #important to add {host_ip} for containers communication
+export http_proxy=${your_http_proxy}
+export https_proxy=${your_http_proxy}
+export NEO4J_URI=${your_neo4j_url}
+export NEO4J_USERNAME=${your_neo4j_username}
+export NEO4J_PASSWORD=${your_neo4j_password} #should match what was used in NEO4J_AUTH when running the neo4j-apoc
+export PYTHONPATH=${path_to_comps}
+export OPENAI_KEY=${your_openai_api_key} #optional, when not provided will use smaller models TGI/TEI
+export HUGGINGFACEHUB_API_TOKEN=${your_hf_token}
+#set additional environment settings
```

Comment:
For all these lines with comments:
Leave one space after `#` and for inline comments leave 2 spaces before `#`.